clients/lsp-yaml.el

```diff
@@ -185,22 +185,27 @@ Set FORCE-DOWNLOADING to non-nil to force re-download the database."
             (alist-get 'schemas (json-read-file lsp-yaml-schema-store-local-db)))))
   (seq-concatenate 'list (list lsp-yaml--built-in-kubernetes-schema) lsp-yaml--schema-store-schemas-alist))
 
-(defun lsp-yaml-set-buffer-schema (uri)
+(defun lsp-yaml-set-buffer-schema (uri-string)
   "Set yaml schema for the current buffer to URI."
   (interactive "MURI: ")
-  (let* ((workspace-path (file-relative-name
+  (let* ((uri (intern uri-string))
+         (workspace-path (file-relative-name
                           (lsp--uri-to-path (lsp--buffer-uri))
                           (lsp-workspace-root (lsp--buffer-uri))))
-         (glob-pattern (concat "/" workspace-path))
+         (glob (concat "/" workspace-path))
          (current-config (cl-assoc uri lsp-yaml-schemas :test 'equal))
```

Comment:
nit: using `assoc` is better since it's C primitive
```suggestion
         (current-config (assoc uri lsp-yaml-schemas))
```

Reply:
nice, thanks